# wheel speeds in ticks per seconds. 
# Used when there are more that two driven wheels.
 
int16[] Speeds